### DISCUSSION TITLE
GLM feature requests

### DISCUSSION THREAD

**Discussion by matthipsey** [AquaticEcoDynamics/GLM3#33] [General]
## 👋 Hi GLM Folk!
  Here is a discussion thread to allow us to keep track and discuss feature requests. As of Apr 2021, v3.2 is being prepared and please post any requests here. 

  To get started, comment below with an introduction  and tell us about what you would like to see!

Best wishes, 
Matt and GLM developers

**Comment by tadhg-moore** [comment]
Would it be possible to include an option to produce daily averaged output? As currently when doing future climate projections I save output every 24 hours and for some lakes the selection of which time to save the output at can influence the signal seen.
